com/uber/kafkaSpraynozzle/KafkaLog.java

```diff
@@ -64,6 +68,11 @@ public void run() {
                 }
             }
             System.out.println(String.format(LOG_FORMAT, this.topic, enqueued, filteredOut, clogged, this.url, posting, postSuccess, postFailure));
+            statsReporter.count("enqueued", enqueued);
+            statsReporter.count("paused", clogged);
+            statsReporter.count("posted", posting);
+            statsReporter.count("postSuccess", postSuccess);
+            statsReporter.count("postFailure", postFailure);
```

Comment:
I completely agree with this approach for at least a first pass. If we did decide to separate logging and statsd from each other, we'd probably want to switch to a full event-driven arch where different mechanisms listen for events emitted by the code and do something with the ones they've registered for.